src/vonage.js

```diff
@@ -13,6 +13,39 @@ const FROM_NUMBER = process.env.FROM_NUMBER;
 
 export const vonage = new Vonage(getVCRAuth());
 
+export const getApplicationNumbers = async () => {
+  const numbers = await vonage.numbers.getOwnedNumbers({
+    applicationId: getVCRAppliationId(),
+  });
+
+  
+  if (Object.keys(numbers).length > 0) {
+    return numbers;
+  }
+
+  return [];
+}
+
+const numberInfoCache = {};
```

Comment:
You don't need to change this for this PR; this is just an FYI. Using `Set` is more performant for cache stuff.